File: ad-hoc/java/3468.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner entrada = new Scanner(System.in);
 
        String idea;
        idea = entrada.nextLine().toLowerCase();

        if (idea.equals("oposicao") || idea.equals("contrariedade")) {
            System.out.println("mas");
        } else if (idea.equals("quantidade") || idea.equals("intensidade")) {
            System.out.println("mais");
        }
    }
}

File: ad-hoc/c/3468.c
#include <stdio.h>
#include <string.h>
#include <ctype.h>

int main() {
    char idea[20];
    scanf("%s", idea);

    // Converter a entrada para letras minúsculas para facilitar a comparação
    for (int i = 0; idea[i]; i++) {
        idea[i] = tolower(idea[i]);
    }

    if (strcmp(idea, "oposicao") == 0 || strcmp(idea, "contrariedade") == 0) {
        printf("mas\n");
    } else if (strcmp(idea, "quantidade") == 0 || strcmp(idea, "intensidade") == 0) {
        printf("mais\n");
    }

    return 0;
}
